### ISSUE TITLE
Cursor flicker

### ISSUE THREAD

**Issue by msorscher** [notepad-plus-plus/notepad-plus-plus#1761]
I've installed the most recent upgrade to Notepad++: v6.9

Now whenever my cursor moves into the Notepad++ frame, it loses its arrow image, and takes on the edit shape (kind of like a capital I with serifs), or the double-headed arrow resize icon.  It flickers and doesn't keep it's shape.

**Comment by rillig-nm** [comment]
Does it happen always? I experience the same behavior, but only after this sequence:

1. Open a file in Notepad++
2. Edit the file, but don't save it
3. Change the file outside of Notepad++
4. Switch back to Notepad++
5. In the Reload question dialog, answer Yes

When moving the mouse cursor now, the cursor flickers. This is especially visible in the title bar, where the cursor flickers between the insert text beam and the arrow.

Notepad++ v7   (64-bit)
Build time : Sep 21 2016 - 22:07:28
Path : C:\Program Files\Notepad++\notepad++.exe
Admin mode : OFF
Local Conf mode : OFF
OS : Windows 10
Plugins : mimeTools.dll NppConverter.dll

**Comment by SinghRajenM** [comment]
Possible duplicate of #2010 , #1018,
Fix has been submitted #2952
